__tests__/app.test.js

```diff
@@ -148,4 +148,71 @@ describe('GET /api/articles/:article_id/comments', () => {
             expect(comments).toHaveLength(0);
         });
     });
+});
+
+describe('PATCH/api/articles/:article_id', () => {
+    it('201: updates the votes of the specified article by the amount sent and returns the updated article.', () => {
+        const item = {inc_votes: -10};
+        return request(app)
+        .patch('/api/articles/1')
+        .send(item)
+        .expect(201)
```

Comment:
for a patch request, i would have the status as a 200 - we are not creating new content here, just amending something that already exists